packages/x-tailwindcss/src/x-tailwind-plugin/components/product-image/index.ts

```diff
@@ -0,0 +1,29 @@
+import { rename } from '@empathyco/x-utils';
+import { TailwindHelpers } from '../../../types';
+import { productImageDefault } from './default';
+import { overlay } from './overlay';
+import { zoom } from './zoom';
+
+/**
+ * Returns the component `input` CSS.
```

Comment:
**typo**: fix component reference in documentation

```suggestion
 * Returns the component `product-image` CSS.
```

Reply:
#shame copy-paste